### PR TITLE
ORBit2: Add package version 2.14.19

### PR DESCRIPTION
**Pull request by Sumynwa**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 
Add package version 2.14.19 in Mariner

###### Change Log  
- add package version 2.14.19
- fix package check section
- lint spec
- update cgmanifest and licenses entry

###### Does this affect the toolchain?  
NO

###### Associated issues  
None

###### Test Methodology

- Pipeline build id: Buddy Build https://dev.azure.com/mariner-org/mariner/_build/results?buildId=270081&view=results

### CODE REVIEW

File: SPECS-EXTENDED/ORBit2/ORBit2.spec [hunk 1]
```diff
@@ -0,0 +1,643 @@
...
+ORBit is a high-performance CORBA (Common Object Request Broker
+Architecture) ORB (object request broker) with support for the
+C language.
+
+This package contains the header files, libraries and utilities
+necessary to write programs that use CORBA technology. If you want to
+write such programs, you'll also need to install the ORBIT package.
+
+%prep
+%setup -q
```
Comment (neha170) [inline SPECS-EXTENDED/ORBit2/ORBit2.spec:64]: Consider using %autosetup

File: SPECS-EXTENDED/ORBit2/ORBit2.spec [hunk 2]
```diff
@@ -0,0 +1,643 @@
...
+%patch0 -p1 -b .multilib
+%patch1 -p1 -b .ref-leaks
+%patch2 -p1 -b .make-j
+%patch3 -p1 -b .deprecated
+
+%build
+%configure --disable-gtk-doc --enable-purify --disable-static --disable-rpath
+make %{?_smp_mflags}
```
Comment (neha170) [inline SPECS-EXTENDED/ORBit2/ORBit2.spec:72]: Consider using %make_build

**Review by neha170 — APPROVED** [review]
Looks good, some minor comments, thanks

File: SPECS-EXTENDED/ORBit2/ORBit2.spec (hunk 1 above)
Comment (Sumynwa) [reply SPECS-EXTENDED/ORBit2/ORBit2.spec]: Done. Updated prep section

File: SPECS-EXTENDED/ORBit2/ORBit2.spec (hunk 2 above)
Comment (Sumynwa) [reply SPECS-EXTENDED/ORBit2/ORBit2.spec]: Done

**Pull request merged**